src/LaravelRequestDocs.php

```diff
@@ -119,15 +119,23 @@ public function filterByMethods($docs, $get, $post, $put, $path, $delete, $head)
 
     public function groupDocs($docs, $group = 'default')
     {
+        if ($group === 'default') {
```

Comment:
better than before, but still not quite there.
![Screen Shot 2023-02-25 at 17 28 30](https://user-images.githubusercontent.com/629055/221347301-918aaa74-35d8-4a0b-9e26-1e77f9468f31.png)


```
    {
        "uri": "api/v1/health",
        "methods": [
            "GET"
        ],
        "middlewares": [
            "api"
        ],
....
        ],
        "group": "api/v1/health",
        "group_index": 0
    },
    {
        "uri": "api/v1/not-health",
        "methods": [
            "GET"
        ],
```



merged but will reopen the issue #143 
Won't push new tag yet.

Reply:
Loop on UI is here

https://github.com/rakutentech/laravel-request-docs/blob/374f4f16cf0d618c3905aa9f5750432e213bcd6b/ui/src/components/Sidebar.tsx#L22-L43